README.md

```diff
@@ -133,6 +133,36 @@ int main(void){
 }
 ```
 
+## Enabling traces in mbed OS 5
+
+To enable traces in mbed OS 5, add the following to your ``mbed_app.json``:
+
+```json
+{
+   "target_overrides": {
+      "*": {
+         "mbed-trace.enable": 1
```

Comment:
The library is inside FEATURE_COMMON_PAL, so shouldn't that have to be added here as well?